BackEndLib/Wchar.h

```diff
@@ -129,7 +129,7 @@ bool isWInteger(const WCHAR* wcz);
 bool isInteger(const char* pcz);
 
 int      _Wtoi(const WCHAR* wsz);
-WCHAR*   _itoW(int value, WCHAR* wcz, int radix);
+WCHAR*   _itoW(int value, WCHAR* wcz, int radix, int bufferLength = 512);
```

Comment:
This is closer.  One more loose end:
In the method definition, we need to clamp bufferLength to 512.  Suggest replacing these magic number instances of 512 with a #define or something even better for ease of maintenance.